README.md

```diff
@@ -489,3 +493,113 @@ ensures that as new modalities are added, and current modalities are extended, n
+|  26   | InvalidTotalTokenSupply           |
+|  27   | MissingTokenID                    |
+|  28   | InvalidTokenIdentifier            |
+|  29   | MissingTokenOwners                |
+|  30   | MissingAccountHash                |
+|  31   | InvalidAccountHash                |
+|  32   | TokenSupplyDepleted               |
+|  33   | MissingOwnedTokensDictionary      |
+|  34   | TokenAlreadyBelongsToMinterFatal  |
+|  35   | FatalTokenIdDuplication           |
+|  36   | InvalidMinter                     |
+|  37   | MissingMintingMode                |
+|  38   | InvalidMintingMode                |
+|  39   | MissingInstallerKey               |
+|  40   | FailedToConvertToAccountHash      |
+|  41   | InvalidBurner                     |
+|  42   | PreviouslyBurntToken              |
+|  43   | MissingAllowMinting               |
+|  44   | InvalidAllowMinting               |
+|  45   | MissingNumberOfMintedTokens       |
+|  46   | InvalidNumberOfMintedTokens       |
+|  47   | MissingTokenMetaData              |
+|  48   | InvalidTokenMetaData              |
+|  49   | MissingApprovedAccountHash        |
+|  50   | InvalidApprovedAccountHash        |
+|  51   | MissingApprovedTokensDictionary   |
+|  52   | TokenAlreadyApproved              |
+|  53   | MissingApproveAll                 |
+|  54   | InvalidApproveAll                 |
+|  55   | MissingOperator                   |
+|  56   | InvalidOperator                   |
+|  57   | Phantom                           |
+|  58   | ContractAlreadyInitialized        |
+|  59   | MintingIsPaused                   |
+|  60   | FailureToParseAccountHash         |
+|  61   | VacantValueInDictionary           |
+|  62   | MissingOwnershipMode              |
+|  63   | InvalidOwnershipMode              |
+|  64   | InvalidTokenMinter                |
+|  65   | MissingOwnedTokens                |
+|  66   | InvalidAccountKeyInDictionary     |
+|  67   | MissingJsonSchema                 |
+|  68   | InvalidJsonSchema                 |
+|  69   | InvalidKey                        |
+|  70   | InvalidOwnedTokens                |
+|  71   | MissingTokenURI                   |
+|  72   | InvalidTokenURI                   |
+|  73   | MissingNftKind                    |
+|  74   | InvalidNftKind                    |
+|  75   | MissingHolderMode                 |
+|  76   | InvalidHolderMode                 |
+|  77   | MissingWhitelistMode              |
+|  78   | InvalidWhitelistMode              |
+|  79   | MissingContractWhiteList          |
+|  80   | InvalidContractWhitelist          |
+|  81   | UnlistedContractHash              |
+|  82   | InvalidContract                   |
+|  83   | EmptyContractWhitelist            |
+|  84   | MissingReceiptName                |
+|  85   | InvalidReceiptName                |
+|  86   | InvalidJsonMetadata               |
+|  87   | InvalidJsonFormat                 |
+|  88   | FailedToParseCep99Metadata        |
```

Comment:
Is CEP99 correct here?

Reply:
I pulled them from https://github.com/casper-ecosystem/cep-78-enhanced-nft/blob/dev/contract/src/error.rs - so we'll need to fix it there as well if they're incorrect. @darthsiroftardis

Reply:
Yeah, thats a mistake thanks for the catch there @davidatwhiletrue

Reply:
Fixed.